#0 building with "default" instance using docker driver

#1 [internal] load build definition from v1_Dockerfile
#1 transferring dockerfile: 511B done
#1 DONE 0.0s

#2 [internal] load metadata for docker.io/library/alpine:latest
#2 DONE 0.6s

#3 [internal] load .dockerignore
#3 transferring context: 2B done
#3 DONE 0.0s

#4 [1/3] FROM docker.io/library/alpine:latest@sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5
#4 resolve docker.io/library/alpine:latest@sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5 0.0s done
#4 sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5 1.85kB / 1.85kB done
#4 sha256:eddacbc7e24bf8799a4ed3cdcfa50d4b88a323695ad80f317b6629883b2c2a78 528B / 528B done
#4 sha256:324bc02ae1231fd9255658c128086395d3fa0aedd5a41ab6b034fd649d1a9260 1.47kB / 1.47kB done
#4 sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6 0B / 3.62MB 0.1s
#4 sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6 3.62MB / 3.62MB 0.2s done
#4 extracting sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6 0.1s done
#4 DONE 0.4s

#5 [2/3] RUN apk add --no-cache python3 &&     python3 -m ensurepip &&     rm -r /usr/lib/python*/ensurepip &&     pip3 install --upgrade pip setuptools &&     if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi &&     if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi &&     rm -r /root/.cache
#5 0.168 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/x86_64/APKINDEX.tar.gz
#5 0.302 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/x86_64/APKINDEX.tar.gz
#5 0.585 (1/17) Installing libbz2 (1.0.8-r6)
#5 0.597 (2/17) Installing libexpat (2.6.2-r0)
#5 0.610 (3/17) Installing libffi (3.4.6-r0)
#5 0.623 (4/17) Installing gdbm (1.23-r1)
#5 0.635 (5/17) Installing xz-libs (5.6.1-r3)
#5 0.649 (6/17) Installing libgcc (13.2.1_git20240309-r0)
#5 0.662 (7/17) Installing libstdc++ (13.2.1_git20240309-r0)
#5 0.703 (8/17) Installing mpdecimal (4.0.0-r0)
#5 0.718 (9/17) Installing ncurses-terminfo-base (6.4_p20240420-r0)
#5 0.732 (10/17) Installing libncursesw (6.4_p20240420-r0)
#5 0.747 (11/17) Installing libpanelw (6.4_p20240420-r0)
#5 0.760 (12/17) Installing readline (8.2.10-r0)
#5 0.775 (13/17) Installing sqlite-libs (3.45.3-r1)
#5 0.802 (14/17) Installing python3 (3.12.3-r1)
#5 1.032 (15/17) Installing python3-pycache-pyc0 (3.12.3-r1)
#5 1.179 (16/17) Installing pyc (3.12.3-r1)
#5 1.191 (17/17) Installing python3-pyc (3.12.3-r1)
#5 1.203 Executing busybox-1.36.1-r29.trigger
#5 1.207 OK: 50 MiB in 31 packages
#5 3.437 error: externally-managed-environment
#5 3.437 
#5 3.437 × This environment is externally managed
#5 3.437 ╰─> 
#5 3.437     The system-wide python installation should be maintained using the system
#5 3.437     package manager (apk) only.
#5 3.437     
#5 3.437     If the package in question is not packaged already (and hence installable via
#5 3.437     "apk add py3-somepackage"), please consider installing it inside a virtual
#5 3.437     environment, e.g.:
#5 3.437     
#5 3.437     python3 -m venv /path/to/venv
#5 3.437     . /path/to/venv/bin/activate
#5 3.437     pip install mypackage
#5 3.437     
#5 3.437     To exit the virtual environment, run:
#5 3.437     
#5 3.437     deactivate
#5 3.437     
#5 3.437     The virtual environment is not deleted, and can be re-entered by re-sourcing
#5 3.437     the activate file.
#5 3.437     
#5 3.437     To automatically manage virtual environments, consider using pipx (from the
#5 3.437     pipx package).
#5 3.437 
#5 3.437 note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
#5 3.437 hint: See PEP 668 for the detailed specification.
#5 3.496 Traceback (most recent call last):
#5 3.496   File "<frozen runpy>", line 198, in _run_module_as_main
#5 3.496   File "<frozen runpy>", line 88, in _run_code
#5 3.496   File "/usr/lib/python3.12/ensurepip/__main__.py", line 5, in <module>
#5 3.496     sys.exit(ensurepip._main())
#5 3.496              ^^^^^^^^^^^^^^^^^
#5 3.496   File "/usr/lib/python3.12/ensurepip/__init__.py", line 284, in _main
#5 3.496     return _bootstrap(
#5 3.496            ^^^^^^^^^^^
#5 3.496   File "/usr/lib/python3.12/ensurepip/__init__.py", line 200, in _bootstrap
#5 3.496     return _run_pip([*args, *_PACKAGE_NAMES], additional_paths)
#5 3.496            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#5 3.496   File "/usr/lib/python3.12/ensurepip/__init__.py", line 101, in _run_pip
#5 3.496     return subprocess.run(cmd, check=True).returncode
#5 3.496            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#5 3.496   File "/usr/lib/python3.12/subprocess.py", line 571, in run
#5 3.496     raise CalledProcessError(retcode, process.args,
#5 3.496 subprocess.CalledProcessError: Command '['/usr/bin/python3', '-W', 'ignore::DeprecationWarning', '-c', '\nimport runpy\nimport sys\nsys.path = [\'/tmp/tmpnsmvkcmg/pip-24.0-py3-none-any.whl\'] + sys.path\nsys.argv[1:] = [\'install\', \'--no-cache-dir\', \'--no-index\', \'--find-links\', \'/tmp/tmpnsmvkcmg\', \'pip\']\nrunpy.run_module("pip", run_name="__main__", alter_sys=True)\n']' returned non-zero exit status 1.
#5 ERROR: process "/bin/sh -c apk add --no-cache python3 &&     python3 -m ensurepip &&     rm -r /usr/lib/python*/ensurepip &&     pip3 install --upgrade pip setuptools &&     if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi &&     if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi &&     rm -r /root/.cache" did not complete successfully: exit code: 1
------
 > [2/3] RUN apk add --no-cache python3 &&     python3 -m ensurepip &&     rm -r /usr/lib/python*/ensurepip &&     pip3 install --upgrade pip setuptools &&     if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi &&     if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi &&     rm -r /root/.cache:
3.496            ^^^^^^^^^^^
3.496   File "/usr/lib/python3.12/ensurepip/__init__.py", line 200, in _bootstrap
3.496     return _run_pip([*args, *_PACKAGE_NAMES], additional_paths)
3.496            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
3.496   File "/usr/lib/python3.12/ensurepip/__init__.py", line 101, in _run_pip
3.496     return subprocess.run(cmd, check=True).returncode
3.496            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
3.496   File "/usr/lib/python3.12/subprocess.py", line 571, in run
3.496     raise CalledProcessError(retcode, process.args,
3.496 subprocess.CalledProcessError: Command '['/usr/bin/python3', '-W', 'ignore::DeprecationWarning', '-c', '\nimport runpy\nimport sys\nsys.path = [\'/tmp/tmpnsmvkcmg/pip-24.0-py3-none-any.whl\'] + sys.path\nsys.argv[1:] = [\'install\', \'--no-cache-dir\', \'--no-index\', \'--find-links\', \'/tmp/tmpnsmvkcmg\', \'pip\']\nrunpy.run_module("pip", run_name="__main__", alter_sys=True)\n']' returned non-zero exit status 1.
------
v1_Dockerfile:3
--------------------
   2 |     
   3 | >>> RUN apk add --no-cache python3 && \
   4 | >>>     python3 -m ensurepip && \
   5 | >>>     rm -r /usr/lib/python*/ensurepip && \
   6 | >>>     pip3 install --upgrade pip setuptools && \
   7 | >>>     if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
   8 | >>>     if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi && \
   9 | >>>     rm -r /root/.cache
  10 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c apk add --no-cache python3 &&     python3 -m ensurepip &&     rm -r /usr/lib/python*/ensurepip &&     pip3 install --upgrade pip setuptools &&     if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi &&     if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi &&     rm -r /root/.cache" did not complete successfully: exit code: 1
Duration: 0:5.12